WireUI/Package.swift

```diff
@@ -38,6 +40,16 @@ let package = Package(
 
         .target(name: "WireFolderPickerUI", dependencies: ["WireReusableUIComponents"]),
 
+        .target(
+            name: "WireIndividualToTeamMigrationUI",
+            dependencies: [
+                .product(name: "WireDomainAPI", package: "WireDomainPackage"),
+                "WireFoundation",
+                "WireReusableUIComponents"
+            ]
+        ),
+        .testTarget(name: "WireIndividualToTeamMigrationUITests", dependencies: ["WireIndividualToTeamMigrationUI"]),
```

Comment:
Please add `WireIndividualToTeamMigrationUITests` to the tesetplan `WireUI/Tests/TestPlans/AllTests.xctestplan`.

Reply:
For some reason the test plan showed all existing targets currently in it as missing. Adding them back has created an absurd diff.